pkg/performance/capabilities/capabilities_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright Antimetal, Inc. All rights reserved.
+//
+// Use of this source code is governed by a source available license that can be found in the
+// LICENSE file or at:
+// https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt
+
+package capabilities
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCapability_String(t *testing.T) {
```

Comment:
I don't see why you need a test like this. If I add a new cap or change the string I have to update this test. so it's really adding anything.